.travis.yml

```diff
@@ -132,10 +132,40 @@ jobs:
         DOCKER_IMAGE_ID: debian-java
         JDK: 11
 
+    - name: "Python on s390x"
+      os: linux
+      arch: s390x
+      env:
+        <<: *global_env
+        ARCH: s390x
+        ARROW_CI_MODULES: "PYTHON"
+        DOCKER_IMAGE_ID: ubuntu-python
+        # Can't enable ARROW_MIMALLOC because of failures in memory pool tests.
+        # Can't enable ARROW_S3 because compiler is killed while compiling
+        # aws-sdk-cpp.
+        DOCKER_RUN_ARGS: >-
+          "
+          -e ARROW_BUILD_STATIC=OFF
+          -e ARROW_FLIGHT=ON
+          -e ARROW_GCS=OFF
+          -e ARROW_MIMALLOC=OFF
+          -e ARROW_ORC=OFF
+          -e ARROW_PARQUET=OFF
```

Comment:
Should we set ARROW_DATASET to ON here? (as that was the original issue of enabling dataset but not parquet giving problems?)

Reply:
It is already on by default.